lib/es5-helpers.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const ChildProcess = require('child_process')
+const Path = require('path')
+const FS = require('fs')
+
+let prefixPath = null
+
+function find(startDir, names) {
```

Comment:
I just published `3.4.0` of helper modules, it's written in es6+ and contains dist files, so it should load in worker now. Means we can use `atom-linter::findFile` (also finds folders) instead of this